### PR TITLE
Update repo's Readme for bannedAPI and publicAPI analyzers

### PR DESCRIPTION
**Pull request by genlu**
https://github.com/dotnet/roslyn-analyzers/issues/2198
#2199

### CODE REVIEW

**Comment by genlu** [comment]
@mavasani @sharwell for review

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
 
+[//]: # (Begin current test results)
+
+|          |Windows Debug|Windows Release|
+|:--------:|:-----------:|:-------------:|
+|**master**|[![Build Status](https://ci.dot.net/job/dotnet_roslyn-analyzers/job/master/job/Windows_NT_Debug/badge/icon)](https://ci.dot.net/job/dotnet_roslyn-analyzers/job/master/job/Windows_NT_Debug/) [![codecov](https://codecov.io/gh/dotnet/roslyn-analyzers/branch/master/graph/badge.svg)](https://codecov.io/gh/dotnet/roslyn-analyzers)|[![Build Status](https://ci.dot.net/job/dotnet_roslyn-analyzers/job/master/job/Windows_NT_Release/badge/icon)](https://ci.dot.net/job/dotnet_roslyn-analyzers/job/master/job/Windows_NT_Release/)|
+
+[//]: # (End current test results)
+
 [![Join the chat at https://gitter.im/roslyn-analyzers/Lobby](https://badges.gitter.im/roslyn-analyzers/Lobby.svg)](https://gitter.im/roslyn-analyzers/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (333fred) [inline README.md]: Consider removing this while you're updating it.

Comment (mavasani) [reply README.md]: Yes, I think someone also filed a bug for this.

Comment (genlu) [reply README.md]: Done

**Pull request merged**